api/types/descriptor.proto

```diff
@@ -15,4 +15,5 @@ message Descriptor {
 	string media_type = 1;
 	string digest = 2 [(gogoproto.customtype) = "github.com/opencontainers/go-digest.Digest", (gogoproto.nullable) = false];
 	int64 size = 3;
+	map<string, string> annotations = 4;
```

Comment:
This is very minor but I think annotations should be 5 to match the OCI `Descriptor` object which has `URLs` as the 4th type. We don't need to add URLs now but if we do in the future it would be nice if the structures were ordered similar.

Reply:
Adjusted.